src/Components/Home/Home.js

```diff
@@ -6,7 +6,7 @@ import { Link, Route, withRouter } from 'react-router-dom';
 import moment from 'moment';
 import NewTaskModal from './NewTaskModal.js';
 import Filter from './Filter';
-import {updateFilteredTask} from '../../actions';
+import Timer from '../Timer/Timer/Timer';
```

Comment:
Timer should not be in the Home component should it? It is rendered from Header, and was removed from Home in my last pull request.

Reply:
Yes, I agree. I think when I did a git merge after your PR that I mistakenly kept the old code. I just did a new commit that removes timer completely from home.js

Reply:
Good catch Christine!